docs/operations/cx-server-operations-guide.md

```diff
@@ -109,30 +123,36 @@ Example:
 > After the completion of the restore operation, it starts the Jenkins server upon user confirmation.
 
 ##### update script
+
 The `cx-server` script can be updated via the `update script` command, if a new version is available.
 ```bash
 ./cx-server update script
 ```
 
 ##### update image
-By default, the Cx Server image defined by `docker_image` in `server.cfg` always points to the newest released version.
```

Comment:
Does not related to the topic of the PR (Document release process)  - like also all the newlines. Blurs the focus of the PR and makes it harder to read.